dtc.c

```diff
@@ -336,8 +336,10 @@ int main(int argc, char *argv[])
 	if (auto_label_aliases)
 		generate_label_tree(dti, "aliases", false);
 
-	if (generate_symbols)
+	if (generate_symbols) {
 		generate_label_tree(dti, "__symbols__", true);
+		generate_labels_from_tree(dti, "__symbols__");
```

Comment:
I'd not immediately sure if it make sense to only do this when `generate_symbols` is set.